### ISSUE TITLE
Duplicate ID

### ISSUE THREAD

**Issue by NTBuddy** [manzinello/mailgo#85]
The Yahoo mail ID is currently m-outlook and should be m-yahoo. Under testing for WCAG it caught this. The WCAG guidance on page parsing and 'id' attributes linked here: https://www.w3.org/WAI/WCAG21/Techniques/html/H93

<a id="m-outlook" href="#mailgo-yahoo" class="m-open m-yahoo">open in <span class="w-500">Yahoo Mail</span></a>

should be

<a id="m-yahoo" href="#mailgo-yahoo" class="m-open m-yahoo">open in <span class="w-500">Yahoo Mail</span></a>

**Comment by manzinello** [comment]
You are completely right. I've fixed it, it will be in the next release of mailgo. Thank you so much!
